enclave-runtime/Cargo.lock

```diff
@@ -1257,23 +1257,55 @@ dependencies = [
  "sha2 0.9.9",
 ]
 
+[[package]]
+name = "ita-sgx-runtime"
+version = "0.8.0"
+dependencies = [
+ "frame-executive",
+ "frame-support",
+ "frame-system",
+ "frame-system-rpc-runtime-api",
+ "pallet-aura",
+ "pallet-balances",
+ "pallet-grandpa",
+ "pallet-parentchain",
+ "pallet-randomness-collective-flip",
+ "pallet-sudo",
+ "pallet-timestamp",
+ "pallet-transaction-payment",
+ "pallet-transaction-payment-rpc-runtime-api",
+ "parity-scale-codec",
+ "scale-info",
```

Comment:
Sanity check: Feature flag is ok because we don't import the evm-pallet in the default sgx-runtime build 👍

Reply:
But we do so in the std part of our worker. Doesn't that go against the sanity check?

Reply:
Hmm, yes, we import the `pallet-evm` in `ita-sgx-runtime` in the top-level cargo.lock .. not sure why though